api/api/routers/view_models/tools_view_models.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
```

Comment:
`view_models` ?

I think we have `schemas` for that purpose (poorly named for sure) but since it's only used in tools_router.py you should just move the defs there ?

Reply:
Right I forgot about the schemas folder. Moved to this folder and renamed to tools_schema.py

Reply:
https://github.com/WorkflowAI/WorkflowAI/pull/204/commits/1505162668d6d199b7c9d8ed4706ea3b394c5734